src/main/java/school/faang/bjs2_74484/Main.java

```diff
@@ -0,0 +1,22 @@
+package school.faang.bjs2_74484;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.TimeUnit;
+
+public class Main {
+    private static final int WAIT_TIME = 10;
+
+    public static void main(String[] args) {
+        MasterCardService masterCardService = new MasterCardService();
+        ExecutorService executor = masterCardService.getSingleThreadExecutor();
+        masterCardService.doAll();
+        executor.shutdown();
```

Comment:
перенести всю логику завершения в сервис

```suggestion
        masterCardService.shutdown();
```